### DISCUSSION TITLE
Customize Login page

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#3028] [Help] [answered]
Id like to display the name of the application instead of **Connexion** label, how can i do ?

![cnx](https://user-images.githubusercontent.com/58070424/177179244-57fc3489-4417-40fc-ad2a-142a5ec6eaf4.jpg)

**Comment by eaziweb95** [comment, marked as answer]
Thanks for your reply

**Reply by studiowizjo** [reply]
There are few possibilities:
1. Change text in translations, but it would be the same for heading, label and any other places if used
2. Publish views, find only the one responsible for this view, make changes and delete the others.

**Reply by eaziweb95** [reply]
I soon tried the second solution but it seems that the login page is linked to Livewire
    'auth' => [
        'guard' => env('FILAMENT_AUTH_GUARD', 'web'),
        'pages' => [
            'login' => \Filament\Http\Livewire\Auth\Login::class,
        ],
    ],

**Reply by eaziweb95** [reply]
the Login.php file to customize is under vendor/filament/filament/src/Http/Livewire/Auth
how can i publish this file ?
php artisan vendor:publish ?????

**Reply by studiowizjo** [reply]
You cannot publish ` \Filament\Http\Livewire\Auth\Login::class` as this is not view file. You could make your own class which extends this file, but there is no sense. Doing like this you should overwrite render method in it to point to your own file. The same, without unnecessary additional class you can achieve with publishing view and overwriting just it. To publish views use the command you wrote. Find correct file, make changes and delete other published files which wasn`t changed.
